src/constraint.cpp

```diff
@@ -493,6 +496,8 @@ void Constraint::MenuConstrain(Command id) {
                             "(symmetric about workplane)\n"));
                 return;
             }
+            // We may remove constraints so remember manually
+            SS.UndoRemember();
             if(c.entityA == Entity::NO_ENTITY) {
                 // Horizontal / vertical symmetry, implicit symmetry plane
                 // normal to the workplane
```

Comment:
This needs to be taken into account as well.
Can be reproduced with "Sketch | Anywhere in 3D" then `Y`
Get the error dialog.
Then you need to undo two times, to undo the last operation before the failed symmetric constraint.